addon/components/column-chart/component.js

```diff
@@ -305,12 +306,12 @@ export default BaseChartComponent.extend({
             labels.remove();
         }
 
-        if (this.get('showMaxMin') && typeof this.get('seriesMaxMin') === 'number' && bars.length > 0) {
-            this.addMaxMinLabels(bars);
+        if (this.get('labelOptions.showDataValues') && typeof this.get('seriesMaxMin') === 'number' && bars.length > 0) {
```

Comment:
Changing `this.get('showDataValues')` => `this.get('labelOptions.showDataValues')` introduces a breaking change in the contract this component has with consumers. We need to keep backwards support for passing in these properties as well as introduce the new options object.

In terms of code edits, do this for both options that are moving to the options hash:
```
const showDataValues = this.get('showDataValues') || this.get('labelOptions.showDataValues');

if (showDataValues && typeof seriesMaxMin === 'number' && bars.length) {
   ...
}
```

When we publish a new major version of this addon, it can drop support for passing `showDataValues` directly in. Maybe we should document that in a changelog, but I'm not certain.

Reply:
ok, will add code for backward compatibility.